wiki/Getting-Started-UIKit.md

```diff
@@ -258,3 +258,23 @@ At a high level we are loading the view controller for testing (similar to befor
 <details>
 It's easy to forget to set the accessibility identifier on the button, please check that first. Second, if you don't call `loadForTesting()` your view controller doesn't make it to the window and the hit testing of `simulateTouch()` will also fail. Finally, make sure the button is visible and tappable on the simulator you are using.
 </details>
+
+## BETA Interoperability With SwiftUI
+You can use your SwiftUI `View`s that are [FlowRepresentable](https://wwt.github.io/SwiftCurrent/Protocols/FlowRepresentable.html) in your UIKit workflows. Start with your `View`
+
+```swift
+struct SwiftUIView: View, FlowRepresentable { // SwiftCurrent
+    weak var _workflowPointer: AnyFlowRepresentable? // SwiftCurrent
+
+    var body: some View {
+        Text("FR2")
+    }
+}
+
+```
+
+Now in your UIKit workflow simply use a HostedWorkflowItem.
+
+```swift
+launchInto(Workflow(HostedWorkflowItem<SwiftUIView>.self)) // SwiftCurrent
```

Comment:
Similar to Getting Started SwiftUI, we have added the necessary imports to this code snippet, as well as a note about importing SwiftCurrent_SwiftUI (This will be necessary until we move HostedWorkflowItem into the UIKit product)